bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+- Judgment calls required from BIP Editors are mostly reassigned either to the audience or the BIP champion
+- Tracking of adoption, acceptance, and community consensus is out of scope for the BIPs repository
+- Layer header is no longer restricted to Standards Track
+- Auxiliary files are only permitted in the corresponding BIP’s subdirectory as nobody used the alternative of labeling
+  them with the BIP number
+- The Comments-URI and Comment-Summary headers are dropped from the preamble, Comments as an aspect of the process are
+  discontinued[^comments]
+- The Discussions-To header is dropped, as it has never been used
+- List of acceptable licenses was reduced to the ones previously used
+- "Other Implementations" sections are discouraged
+
+## Copyright
+
+This BIP is licensed under the [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+## Related Specifications
+
+- [BIP 1: BIP Purpose and Guidelines](bip-0001.mediawiki)
+- [BIP 2: BIP Process, revised](BIP-0002.mediawiki)
+- [BIP 123: BIP Classification](BIP-0123.mediawiki)
+* [RFC 7282: On Consensus and Humming in the IETF](https://tools.ietf.org/html/rfc7282)
+
+## Rationale
+
+[^astroturfing]: **What does it mean to be focused on Bitcoin the currency?**  
+    Proposals to astroturf on the Bitcoin network to store data, bootstrap their own consensus mechanism, or facilitate
+    another currency are not on-topic.
+[^comments]: **Why were comments, Comments-URI, and Comment-Summary removed from the process?**  
+    The comments feature saw marginal use. Hardly any BIPs received more than two comments and this led to many
+    situations in which the opinion of a single comment or two commenters ended up sourcing the comment summary. While
+    some of those comments may have been representative of broadly held opinions, it also overstated the importance of
+    individual comments. As collecting feedback in this accessible fashion failed, the new process puts the onus back on
+    the audience to make their own evaluation.
+[^OtherImplementations]: **What is the issue with "Other Implementations" sections in BIPs?**  
+    In the past, some BIPs had "Other Implementations" sections that caused frequent change requests to existing BIPs.
+    This put an onus on the BIP champions, and frequently lead to lingering pull requests due to the corresponding BIPs’
+    champions no longer participating in the process. Many of these alternative implementations eventually became
+    unmaintained or were low-quality to begin with. Therefore, "Other Implementations" sections are heavily discouraged.
+[^champ]: **Why do we prefer 'Champion' over 'Author'?**  
+    Sometimes BIP authors discontinue work on BIPs or leave Bitcoin development. In that case another proponent may
+    adopt the BIP as a new owner, even if they are not the original author.
```

Comment:
"proponent" seems a much better term for the person behind a proposal than champion

Reply:
The general feedback seems to be that the obvious term is author, so I have changed the document to that effect